.env.example

```diff
@@ -1,8 +1,7 @@
-SECRET_KEY=my_secret_key
+SECRET_KEY= replace with your secret key
```

Comment:
What do you think about generating the secret as part of the [`postCreateCommand`](https://github.com/github/codespaces-django/blob/a1f4ea8bdcdd3433e06b7e42b5dd5f38382402b8/.devcontainer/devcontainer.json#L8), so that devs don't have to manually do anything?

Reply:
i taught of doing this but i feel  incase of doing a cloud uploading must time people need to document or change their secret keys reason i added it as script